content/en/localstack/usage-tracking.md

```diff
@@ -0,0 +1,117 @@
+---
+title: LocalStack usage tracking
+weight: 75
+description: >
+  Understand what data LocalStack collects and how you can opt out of usage tracking.
+---
+
+## Overview
+
+For API Key activations, we track the timestamp and the API key. We need to do this to make CI credits work. It is tracked regardless of whether the user disables event tracking since we collect this in the backend, not the client.
+
+## LocalStack usage statistics
+
+For pro users, most of the information is collected to populate the "stack insights" dashboard. Collecting basic anonymized usage of AWS services helps us better direct engineering efforts to services that are used the most or cause the most issues.
+
+### Session information
+
+The current usage event collection on the client side includes:
+
+- A randomly generated ID pertaining to the session.
```

Comment:
nit: We could unify the bullet points below, and either add a dot `.` at the end or remove it from all items.